CsStat.Web/assets/components/base-layout/scss/base-layout.scss

```diff
@@ -25,4 +26,26 @@
             margin: 0;
         }
     }
+
+    &__footer {
+        background: lighten($color-dark-blue, 10%);
+        padding: 20px 0 0;
+        color: $color-white;
+    }
+
+    &__footer-copyright {
+        padding: 8px 20px;
+        display: flex;
+        justify-content: space-between;
+        background: lighten($color-dark-blue, 5%);
+
+        @include mobile-only() {
+            flex-direction: column;
+            align-items: center;
+        }
+    }
+
+    &__footer-copyright a {
```

Comment:
It's possible to use a BEM element for link here instead of the DOM element's selector.